apps/docs/licensing.md

```
@@ -0,0 +1,18 @@
+# Licensing
+
+Task Master is licensed under the MIT License with Commons Clause. This means you can:
+
+## ✅ Allowed:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)